src/elements/diagnostics/BeamMonitor.cpp

```diff
@@ -248,13 +248,13 @@ namespace detail {
         auto d_ui = io::Dataset(dtype_ui, {np});
 
         // openPMD 1.* needs "seconds" here, but we fake it as "s"
-        iteration.setTime(ref_part.s);
+        iteration.setTime(ref_part.s.value());
```

Comment:
Interesting. Should we just use
```suggestion
        iteration.setTime(ref_part.t.value());
```
?

(Should be a separate PR, just noticed here...)